src/ui/screens/PermissionGrantingPrompt.tsx

```diff
@@ -8,10 +8,11 @@ import {Caption, Headline, Button, useTheme, Subheading, Icon} from '@ui/library
 import {Layout} from '@ui/components/Layout';
 
 type Props = {
-  skipPnPermission: () => void;
+  skipPnPermission?: () => void;
+  isSkip: boolean;
 };
 
-function PermissionGrantingPromptScreen({skipPnPermission}: Props) {
+function PermissionGrantingPromptScreen({skipPnPermission, isSkip}: Props) {
```

Comment:
And here you can set a default value like `isSkip = false`. So when `isSkip` is `undefined` it will take the default value.